pubsublite/internal/wire/rpc.go

```diff
@@ -164,18 +164,64 @@ func newPartitionAssignmentClient(ctx context.Context, region string, opts ...op
 	return vkit.NewPartitionAssignmentClient(ctx, options...)
 }
 
-func addTopicRoutingMetadata(ctx context.Context, topic topicPartition) context.Context {
-	md, _ := metadata.FromOutgoingContext(ctx)
-	md = md.Copy()
-	val := fmt.Sprintf("partition=%d&topic=%s", topic.Partition, url.QueryEscape(topic.Path))
-	md[routingMetadataHeader] = append(md[routingMetadataHeader], val)
-	return metadata.NewOutgoingContext(ctx, md)
+const (
+	routingMetadataHeader    = "x-goog-request-params"
+	clientInfoMetadataHeader = "x-goog-pubsub-context"
+
+	languageKey     = "language"
+	languageValue   = "GOLANG"
```

Comment:
This should probably just be Go? Also you can tell a call came from a Go library already by looking at metadata in the `x-goog-api-client` header. If it is Go there will be a gl-go value present with the version of Go that is being used.

Reply:
Thanks Cody. These values are for a pub/sub specific header x-goog-pubsub-context. I'll check if it's documented anywhere. "GOLANG" was already defined for Go and recognized on the server. I agree that it should have just been Go.

The servers haven't been inspecting x-goog-api-client, but I think it would be a good idea to use it as backup, for when the pubsub header is missing.

Reply:
Ack